README.md

```diff
@@ -245,12 +245,13 @@ kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/
 Note the `match` field, which defines the scope of objects to which a given constraint will be applied. It supports the following matchers:
 
    * `kinds` accepts a list of objects with `apiGroups` and `kinds` fields that list the groups/kinds of objects to which the constraint will apply. If multiple groups/kinds objects are specified, only one match is needed for the resource to be in scope.
+   * `scope` accepts `*`, `Cluster`, or `Namespaced` which determines if cluster-scoped and/or namesapced-scoped resources are selected. (defaults to `*`)
    * `namespaces` is a list of namespace names. If defined, a constraint will only apply to resources in a listed namespace.
    * `excludedNamespaces` is a list of namespace names. If defined, a constraint will only apply to resources not in a listed namespace.
    * `labelSelector` is a standard Kubernetes label selector.
    * `namespaceSelector` is a standard Kubernetes namespace selector. If defined, make sure to add `Namespaces` to your `configs.config.gatekeeper.sh` object to ensure namespaces are synced into OPA. Refer to the [Replicating Data section](#replicating-data) for more details.
 
-Note that if multiple matchers are specified, a resource must satisfy each top-level matcher (`kinds`, `namespaces`, etc.) to be in scope. Each top-level matcher has its own semantics for what qualifies as a match. An empty matcher is deemed to be inclusive (matches everything).
+Note that if multiple matchers are specified, a resource must satisfy each top-level matcher (`kinds`, `namespaces`, etc.) to be in scope. Each top-level matcher has its own semantics for what qualifies as a match. An empty matcher is deemed to be inclusive (matches everything). Also understand `namespaces`, `excludedNamespaces`, and `namespaceSelector` will match on cluster scoped resources which are not namespaced. To avoid this adjust the `scope` to `Namespaced`.
```

Comment:
> `namespaces`, `excludedNamespaces`, and `namespaceSelector` will match on cluster scoped resources which are not namespaced 

In a scenario where I provide `pod` for `kinds`, I would expect `namespaces`, `excludedNamespaces`, and `namespaceSelector` matchers to also match on `Namespaced` for pod resources. This is how it works today. With this new feature, would the default `scope` of `*` achieve that? This is a bit confusing.

Reply:
Since `scope` is at the top level, what if I set `scope` as `Cluster` and `pod` for `kinds`, would that result in no match since all matchers have to be satisfied?

Reply:
Yes, with the scope set to `•` (or `Namespaced`) Pods would be considered for a match. The only scenario pods would be ignored is if the `scope` was set to `Cluster`.

The main reason I've justified adding `scope` is to support always matching `namespaces`, `excludedNamespaces`, and `namespaceSelector` when the resource doesn't have namespace. I believe this is the right direction because this is how `namespaceSelector` works on components native to Kubernetes.

I'm open to changing the wording on the docs though. I understand it's kinda confusing at a glance.